강지은/src/CounterPage.jsx

```diff
@@ -0,0 +1,22 @@
+
+import React, { useState } from 'react';
+
+
+function CounterPage() {
+    const [count, setCount] = useState(0) 
+  return (
+    <div>
+      <div>
+        count: {count}
+      </div>
+      <button onClick={() =>{
+      setCount(count +1)
+      }}>+</button>
```

Comment:
버튼에 직접 핸들러 함수를 넣는 것도 좋지만, 따로 핸들러 함수를 선언해서 전달하면 가독성도 높아지고 UI와 로직을 분리할 수 있어 유지보수에 더 좋다고하더라구요! 한 번 적용해보시는 건 어떠신가요?
```suggestion
const increase = () => {
  setCount((prev) => prev + 1);
};
// ...
      <button onClick={increase}>+</button>
```